src/app/orderHistory/styles.ts

```diff
@@ -0,0 +1,45 @@
+// styles.ts (for styled-components)
+import styled from 'styled-components';
+import NavBar from '../../components/NavBarFolder/NavBar';
+
+import Footer from '../../components/FooterFolder/Footer';
+
+export const FooterMoved = styled(Footer)`
+  left: 0;
+`;
+
+export const NavBarMovedUP = styled(NavBar)`
+  position: relative;
+`;
+
+export const OutterBox = styled.div`
+  width: 900px;
+  margin: 0 auto; // This will center the OutterBox
+`;
+
+export const OrderHistoryContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 10px;
+  background: var(--White, #fff);
+  box-shadow: 0px 1px 4px 1px rgba(0, 0, 0, 0.2);
+  width: 800px; // Width of the outer box
+  height: 700px;
+  overflow: scroll;
+  margin-top: 10px;
+  padding: 0; // Ensure there's no padding pushing the internal boxes inward
+`;
+
+export const OrderHistoryBox = styled.div`
+  width: 100%; // Make internal box take full width of the container
+  margin: 20px 0; // Keep vertical spacing between orders
+  padding: 30px; // Inner spacing for content inside the box
+  border-radius: 8px;
+  background: #fff;
```

Comment:
same here

Reply:
a little confused on why both orderhistorybox and orderhistorycontainer are both flex boxes that are scrollable. What is the difference and do you mind changing the name to something that would help me understand the difference since in some cases a box is a container